Integrations/Slack/Slack.py

```diff
@@ -187,6 +197,54 @@ def set_to_latest_integration_context(key: str, value, wait: bool = False):
     demisto.setIntegrationContext(integration_context)
 
 
+def send_slack_request_sync(client: slack.WebClient, method: str, http_verb: str = 'POST', file: dict = None,
+                            body: dict = None) -> SlackResponse:
+    while True:
+        try:
+            if http_verb == 'POST':
+                if file:
+                    response = client.api_call(method, files={"file": file}, data=body)
```

Comment:
why for post we use `data` and for other we use `json`?

Reply:
That's how they send files